.github/workflows/cpaas.yml

```diff
@@ -57,3 +57,25 @@ jobs:
     - init
     - build-collector-full
     secrets: inherit
+
+  notify:
+    runs-on: ubuntu-latest
+    if: always() && contains(join(needs.*.result, ','), 'failure') && github.event_name == 'schedule'
+    needs:
+    - init
+    - sync-drivers-and-support-packages
+    - build-collector-slim
+    - build-collector-full
+    - integration-tests
+    steps:
+      - name: Slack notification
+        uses: rtCamp/action-slack-notify@v2
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_COLLECTOR_NOTIFICATIONS_WEBHOOK }}
+          SLACK_CHANNEL: collector-notifications
+          SLACK_COLOR: ${{ job.status }}
```

Comment:
I think this will be the wrong colour, because this job (`notify`) has not failed. I think in this case it's better to hard-code it to 'failed' (IIRC) since we know we're only triggering when the `needs` have failed

Reply:
Changed it to `failure` which is the one I found for the integration tests notification.